passpwn

```diff
@@ -38,11 +39,54 @@ function main() {
 		entry=${escaped_entry%\"}
 		entry=${entry#\"}
 		entry=${entry%.gpg}
+		verbose_log "Checking password file ${entry}"
 
-		# Check the pass entry.
-		check_password "${entry}"
+		# Check the pass entries:
+		if [ "$FULL_FILE" = true ] ; then
+			while read p; do
+				check_password "${p}"
+			done <<< "$(pass -- "${entry}")"
+		else
+			check_password "$(pass -- "${entry}" | head -n 1)"
+		fi
 	done
 }
 
+# Verbose logging:
+function verbose_log() {
+	if [ "$VERBOSE" = true ] ; then 
+		echo "VERBOSE: ${1}"
+	fi
+}
+
+# Help:
+function usage() { 
+	echo "Usage: $0 [-h] [-v] [-f]" 1>&2; 
+	exit 1; 
```

Comment:
What do you think about returning 0 here? I looked at `vim -h`, `gdb -h` and `cmake -h` and they seem to all return 0 for help.